.github/workflows/build.yml

```diff
@@ -0,0 +1,46 @@
+name: Build images
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - 'v*'
+  pull_request:
+    branches:
+      - master
```

Comment:
The way I understand this config we will get tagged images for all pushes to `master`, `v*` tags, and pull requests against master (like this one). According to [the docs](https://github.com/docker/metadata-action#latest-tag) the `latest` tag will be generated as well when we push a `v*` tag.

Reply:
We might not want a `master` image since we pretty much add a `v*` tag to everything that gets pushed to mater. Let me know what you think.